### PR TITLE
[DC-2964] load ad hoc file from RDR bucket

### PR DESCRIPTION
**Pull request by lrwb-aou**
* ability to load an ad hoc file from the RDR bucket
* Will require the user to impersonate a service account with the access
* will require a json schema file
* should be able to load any file from that bucket with no (or minimal) changes needed, even if it is the first time we have seen the file

### CODE REVIEW

File: data_steward/tools/load_rdr_file.py [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+                                        validate_file_exists,
+                                        validate_email_address,
+                                        validate_bucket_filepath)
+from gcloud.bq import BigQueryClient
+from common import CDR_SCOPES
+
+LOGGER = logging.getLogger(__name__)
+
+
+def load_rdr_table(client: BigQueryClient, src_uri: str, table_id: str,
```
Comment (ksdkalluri) [inline data_steward/tools/load_rdr_file.py:19]: @lrwb-aou I understand this is to load a single file, do you think If we can expand this script to load multiple files from a directory?  If that will be too much of a scope creep we can create a new script to add that functionality.

Comment (lrwb-aou) [reply data_steward/tools/load_rdr_file.py:19]: @ksdkalluri I think we can expand it to do that, but would suggest making it a new PR (not necessarily a new script).  Right now, we try to load all the files they send us that we know about with the import rdr omop script.  This script allows our teams to use the bucket to pass data from their prod env to our prod env without downloading the data.  Hopefully, we don't have this happen too often. 🤞 

I think the directory import may be useful, but would like to do that as an additional PR (primarily because I want to enforce useful schemas with valid column descriptions).  Thoughts?

Comment (ksdkalluri) [reply data_steward/tools/load_rdr_file.py:19]: My only concern was, if RDR wants to send us multiple files in the same directory that are not related to the rdr submission white-listed files, we would have to run this script for each file in the directory which will become a manual repetitive process. Since we are not expecting such transfers from RDR anytime soon we can currently go ahead with this scope. We can expand on this to load multiple files if we expect more such transfers in future.

**Review by ksdkalluri — APPROVED** [review]
This is good to merge, thanks for adding all the parameter validators, it'll be helpful for us to add new ones to the validators file in the future.  All the tests were helpful in testing and understanding the code.

**Comment by lrwb-aou** [comment]
Hopefully, if they are going to send us multiple files and it's not an ad hoc situation, we will have a json schema file built for that.  🤞 

If sending data that we don't have a schema for becomes the norm, we've got an SOP problem.

File: data_steward/tools/load_rdr_file.py (hunk 1 above)
Comment (lrwb-aou) [reply data_steward/tools/load_rdr_file.py:19]: Hopefully, if they are going to send us multiple files and it's not an ad hoc situation, we will have a json schema file built for that. 🤞

If sending data that we don't have a schema for becomes the norm, we've got an SOP problem.
